CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add Mimir Alertmanager datasource
 
+### Changed
+
+- Removed organization OwnerReference on grafana-user-values configmap, this fixes an issue where the configmap is removed when the last organization is deleted which prevent Grafana from starting.
```

Comment:
But why would the giantswarm organization ever be removed?

Reply:
I don't know why, but as explained in the PR description, I've seen it happening :grimacing: 
I suspect it may happen during grafana app upgrades. :shrug: